packages/charts/chart-utilities/src/PlotlySchemaConverter.ts

```diff
@@ -72,7 +72,8 @@ export const isDateArray = (data: Datum[] | Datum[][] | TypedArray | undefined):
 };
 
 export const isNumberArray = (data: Datum[] | Datum[][] | TypedArray | undefined): boolean => {
-  return isArrayOfType(data, isNumber);
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  return isArrayOfType(data, (value: any): boolean => typeof value === 'number' || value === null);
```

Comment:
should we check for nan explicitly.

Reply:
JSON doesn't support NaN or other non-finite numbers, so [this line](https://github.com/microsoft/fluentui/blob/4ff1a6183fb6a6f2c7ae91304886eca90ac15966/packages/charts/chart-utilities/src/DecodeBase64Data.ts#L100) in the decodeBase64Fields function converts them to null during successive renders. And Plotly may also produce null data points, so this check allows null values to pass through so that the chart-specific adapters can handle them appropriately.